pkg/cloudprovider/cloudprovider.go

```diff
@@ -246,6 +246,27 @@ func getTags(ctx context.Context, nodeClass *v1.EC2NodeClass, nodeClaim *karpv1.
 	}), staticTags)
 }
 
+func (c *CloudProvider) RepairPolicies() []cloudprovider.RepairPolicy {
```

Comment:
If these are all common to core kubernetes, do these belong in `kubernetes-sigs/karpenter` and then those policies could be extended?

Reply:
We don't want to enforce condition on cloud providers or prescribe the tolerationDuration amount for a cloud provider for this alpha feature.